api/v1beta1/machineset_types.go

```diff
@@ -189,8 +189,8 @@ func (m *MachineSet) Validate() field.ErrorList {
 	if err != nil {
 		errors = append(errors, field.Invalid(fldPath.Child("selector"), m.Spec.Selector, "invalid label selector."))
 	} else {
-		labels := labels.Set(m.Spec.Template.Labels)
-		if !selector.Matches(labels) {
+		lbls := labels.Set(m.Spec.Template.Labels)
+		if !selector.Matches(lbls) {
```

Comment:
Should we keep the current names instead? In general we should always strive for readability over anything else, would you be opposed to keep things as they are given that this hasn't been a huge issue?

Reply:
There's definitely a loss in readability here, I could improve the substitutions used - fewer short names more descriptive names - or we could drop this. It does seem like it could be an issue though - even if it hasn't been in the past - because clashes like these are very hard to catch in manual review.